src/passes/OptimizeInstructions.cpp

```diff
@@ -151,12 +151,16 @@ struct OptimizeInstructions
   : public WalkerPass<
       PostWalker<OptimizeInstructions,
                  UnifiedExpressionVisitor<OptimizeInstructions>>> {
-  bool isFunctionParallel() override { return true; }
-
```

Comment:
please don't change the order of these, the previous order was consistent across the codebase.

Reply:
Got it. Reverted!